_rules/sequentially-focusable-element-has-visible-focus-oj04fd.md

```diff
@@ -0,0 +1,181 @@
+  authors:
+    - Jean-Yves Moyen
+---
+
+## Applicability
+
+The rule applies to any element which is part of [sequential focus navigation][] in a document with at least two [focusable][] elements.
+
+## Expectation
+
+For each target element, there is at least one device pixel inside the [scrolling area][] of the [viewport][] whose color is different when the element is [focused][] from when it is not.
+
+## Assumptions
+
+This rule assumes that documents with only one focusable element automatically satisfies [Success Criterion 2.4.7 Focus Visible][sc247], as stated in [Understanding Success Criterion 2.4.7: Focus Visible][usc247]: "if there is only one keyboard actionable control on the screen, the success criterion would be met".
+
+## Accessibility Support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+Default styling in most user agents is providing perfectly valid focus indication for focusable elements (even those that are not focusable by default), as shown in Passed Examples 1 and 2. Many examples in this rule need to **remove** that indicator in order to illustrate various situations. This is bad practice and should normally be avoided.
+
+WCAG does not have any requirement of how big or small focus indicator should be, or how far or near from the [focusable][] element it should be. Thus it is possible to pass this rule and [Success Criterion 2.4.7 Focus Visible][sc247] with barely perceptible changes at the other end of the page. That would however still be an accessibility issue. WCAG 2.2 includes [Success Criterion 2.4.11 Focus Appearance (Minimum)][sc2411] and [Success Criterion 2.4.12 Focus Appearance (Enhanced)][sc2412] specifying how big the focus indicator should be. All Examples in this rule avoid focus indications that are too small, too far away, too weak, …
+
+WCAG has no clear requirement of unicity of the focus indicator for each [focusable][] element. Therefore, this rule can pass even if several focus indicators are identical. Such a situation may nonetheless cause confusion and all Examples in this rule avoid it.
+
+- [Success Criterion 2.4.7 Focus Visible][sc247]
+- [Success Criterion 2.4.11 Focus Appearance (Minimum)][sc2411]
+- [Success Criterion 2.4.12 Focus Appearance (Enhanced)][sc2412]
+- [Understanding Success Criterion 2.4.7: Focus Visible][usc247]
+- [Understanding Success Criterion 2.4.11: Focus Appearance (Minimum)][usc2411]
+- [Understanding Success Criterion 2.4.12: Focus Appearance (Enhanced)][usc2412]
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The first [focusable][] element is part of [sequential focus navigation][]. It has an outline when it is [focused][] (due to default User Agent's styling). The second [focusable][] element is not applicable because it has been removed from [sequential focus navigation][] due to the negative value for the `tabindex` attribute. Its presence is nonetheless enough to make the first one applicable.
```

Comment:
```suggestion
The first [focusable][] element is part of [sequential focus navigation][]. It has an outline when it is [focused][] (due to default User Agent's styling). The second [focusable][] element is not applicable because it has been removed from [sequential focus navigation][] due to the negative value for the `tabindex` attribute. Its presence is nonetheless enough to make the rule applicable to the first one.
```